waspc/src/Wasp/Generator/Valid.hs

```diff
@@ -0,0 +1,13 @@
+module Wasp.Generator.Valid where
```

Comment:
It is anti pattern to blidnly export everthing from the module in Haskell.
Instead, use `()` and explicitly list in it what you are exporting.